src/components/ui/TimeTrackerHeader.tsx

```diff
@@ -105,6 +106,7 @@ export default function TimeTrackerHeader({ id, workspaceName }: TimeTrackerHead
             if (stopTimerAPI.fulfilled.match(result)) {
                 reset()
                 setSelectedProjectId(null)
+                setResetProjectTrigger(prev => !prev) // Toggle to trigger TrackerInput reset
                 dispatch(getUserTimeLogs(id!))
```

Comment:
can we have these logics done in the extraReducer run after the `stopTimerAPI` AsyncThunk is fullfilled in the `timeLogsSlice`? or is there a reason why it's not there?